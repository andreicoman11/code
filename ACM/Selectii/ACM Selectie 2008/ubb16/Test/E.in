30
30
-115 281
11 295
-9 301
61 300
124 279
84 288
-12 292
-48 287
94 285
-40 296
-33 308
-62 286
5 304
-1 290
97 278
54 304
131 267
-37 301
-107 277
-5 303
-4 298
138 264
-104 280
103 276
45 290
76 285
-46 306
-124 284
82 293
-131 269
30
80 296
-56 288
-29 291
41 306
139 271
50 285
92 278
-62 294
-52 295
39 293
102 276
-150 268
-41 301
-103 274
-6 298
-74 288
-49 297
-130 271
-31 287
-122 261
-96 289
-31 302
-28 294
-122 273
-74 295
-57 302
43 292
-16 290
-140 270
-102 292
30
45 304
-128 260
80 287
80 286
-99 279
-119 274
-27 297
141 266
138 257
98 282
-106 276
-33 293
114 273
-72 289
-110 291
105 279
90 279
92 291
-47 287
-76 292
-78 286
-76 300
108 272
63 303
-77 279
53 287
-41 306
124 272
-144 262
76 287
30
-52 289
106 287
-2 293
-115 288
88 293
57 289
-17 300
-32 298
-23 294
-13 291
-91 278
-42 289
-120 274
91 297
-20 306
128 284
137 280
97 281
-58 286
16 307
-87 294
75 291
81 296
67 283
-47 295
-36 295
-55 302
21 308
-26 309
105 277
30
124 279
15 301
27 303
-129 267
-73 296
31 293
-101 284
2 290
-109 284
-92 284
-113 281
152 263
123 280
55 294
22 310
113 277
-40 301
27 300
33 294
132 269
-22 301
14 298
113 268
94 284
133 279
-44 298
-41 300
-102 274
-130 261
73 300
30
12 306
-115 281
30 293
-138 264
109 283
-146 264
49 290
-149 256
112 282
99 279
-80 283
3 291
-14 304
90 290
88 280
-75 298
102 290
-31 298
-112 285
83 284
82 289
-122 280
151 268
69 291
-117 268
132 266
143 271
-59 299
137 264
-21 294
30
121 283
67 303
81 293
141 271
114 283
-102 293
141 260
-126 265
-139 280
82 301
-2 305
150 269
-34 294
-26 300
16 294
-67 294
-26 299
-93 284
-64 289
-45 287
-7 310
-16 295
-122 278
-139 256
-31 292
-62 290
-50 293
-79 289
131 264
114 268
30
-45 293
-123 260
20 307
133 272
11 306
-69 287
130 278
10 301
42 297
126 265
122 285
109 283
-21 302
53 291
14 296
120 273
-127 281
134 260
-78 283
128 273
71 294
-143 266
74 287
36 304
23 292
123 279
137 257
102 280
-146 256
-53 305
30
-37 289
-33 303
87 297
-63 298
-153 261
-49 300
85 280
114 280
-20 305
62 294
128 269
-34 305
16 306
34 300
-140 272
92 295
11 306
-140 268
83 284
-12 300
-26 307
58 294
-78 298
-36 307
-81 296
-90 285
37 302
-3 310
-54 304
-155 256
30
70 295
-98 294
-144 265
-147 252
28 291
-102 274
-117 267
-85 295
-94 292
-83 289
-149 263
42 293
-108 280
-8 304
119 280
124 277
45 299
46 291
-50 289
104 273
59 297
-13 298
68 290
-127 274
-33 302
-75 297
122 277
8 299
122 260
57 294
30
145 270
87 283
71 291
136 261
-140 269
13 295
23 308
142 266
146 266
-107 279
-90 285
50 296
18 300
50 295
117 280
-100 285
-91 289
12 297
-32 299
66 292
-114 282
50 292
71 300
-119 278
126 269
-69 299
-90 289
148 269
-46 294
77 286
30
-91 284
-69 290
-87 293
-53 301
-92 281
-46 299
-145 252
8 308
30 295
-145 255
-39 288
-67 284
-84 294
34 309
85 282
0 310
-88 277
40 296
51 290
-87 288
77 286
-73 291
105 288
54 293
24 292
-123 285
-106 271
48 297
-90 288
-109 276
30
38 294
-37 294
-45 293
-22 297
90 293
-22 294
-25 298
104 275
-86 284
113 271
-138 263
96 293
128 275
-139 273
-100 279
-117 277
65 295
-73 295
106 281
-59 297
104 292
83 278
6 297
80 295
110 273
57 302
47 295
-62 293
26 297
-130 277
30
-113 277
93 276
-93 284
31 291
-72 288
133 267
-57 299
-139 263
-3 300
-32 296
129 263
14 300
-105 277
72 284
114 270
-105 287
8 302
-3 305
-137 263
-19 304
-87 280
-79 283
114 281
37 296
-62 296
90 283
154 260
-10 300
-66 300
-139 260
30
-79 289
-40 304
74 293
-141 264
60 290
53 299
-81 293
43 303
-3 293
-15 293
-115 283
130 280
-131 270
73 294
10 290
-13 291
-52 291
71 298
-19 297
98 279
44 306
-13 300
49 287
132 271
-65 296
70 298
44 290
80 281
151 251
-147 262
30
33 293
-47 303
111 282
-10 293
35 295
-98 279
-133 275
43 288
-56 292
-19 301
-72 289
30 302
-96 289
-63 283
38 290
-99 278
134 273
-54 292
132 268
28 294
56 304
-106 282
-127 262
-45 305
-2 301
-56 289
16 297
-15 294
-135 270
138 259
30
-8 302
89 290
126 272
132 268
22 305
-111 267
-15 301
-49 297
-18 295
64 291
-117 268
27 294
-94 292
-35 298
-31 293
-69 303
-35 292
-40 295
-92 279
100 287
37 287
10 302
138 263
81 284
91 295
119 278
34 296
-8 294
-31 298
-78 296
30
-105 293
-24 304
26 299
-36 292
12 307
100 293
-75 289
12 298
-56 289
-80 287
142 268
134 262
-19 298
54 297
-90 293
119 269
21 297
95 287
103 289
-34 287
-87 295
-93 280
-111 268
11 302
122 267
41 296
-152 257
78 289
44 294
-110 286
30
-105 274
-118 279
10 299
72 292
75 283
98 289
-71 292
-29 294
109 285
-99 276
-84 295
-1 302
107 273
-4 310
92 284
-55 294
-36 291
53 286
-103 278
-114 280
-85 298
-143 267
-55 293
133 281
-8 297
-19 302
134 273
-2 304
-137 267
56 298
30
-21 293
-49 294
-90 278
-57 286
-115 270
129 275
11 307
19 303
22 308
8 296
111 287
-135 261
81 283
-2 309
74 289
25 299
50 294
-119 266
37 293
81 295
-11 303
139 264
48 299
-143 262
-144 259
-86 290
-111 276
-62 285
59 294
-54 293
30
-95 291
-15 296
128 273
28 291
101 286
85 293
16 294
39 303
95 287
10 299
17 297
134 266
46 304
-96 295
-156 253
-127 273
7 290
-159 250
62 299
109 279
-70 297
118 268
47 304
33 297
40 291
53 302
34 308
-44 295
-34 303
97 291
30
33 297
95 283
-3 306
143 260
3 292
-149 266
137 265
8 292
-138 275
-15 295
-156 256
92 282
44 303
-129 269
97 290
-10 300
-43 291
-108 289
-38 298
135 259
-110 274
-114 272
-81 282
-110 271
-123 282
-134 269
71 298
106 284
67 292
-56 300
30
-134 268
28 302
-30 304
-122 278
-19 289
-71 292
102 283
-72 295
76 283
-101 286
-108 290
116 282
-104 287
-9 295
69 289
120 281
136 261
-143 259
-3 302
37 304
63 301
3 305
16 291
36 290
-8 305
-43 289
-16 293
-20 292
-115 286
-124 271
30
-141 265
109 271
-31 291
64 291
23 291
-63 299
-28 288
55 303
130 271
2 303
-89 279
13 299
94 287
-151 252
-46 291
-116 272
1 300
4 303
115 278
103 272
-124 268
37 293
17 292
67 300
45 297
-52 298
-146 253
-42 303
64 297
-43 291
30
54 289
-26 299
41 298
-6 304
60 294
146 259
119 280
-35 303
71 301
69 286
128 272
-61 287
-51 304
108 270
-138 259
-61 302
139 261
-144 271
-56 295
139 267
-94 277
59 289
-58 296
-25 302
-54 296
20 294
-21 305
2 309
-61 284
46 307
30
56 301
-41 291
-36 298
48 293
14 289
93 285
-108 279
-75 292
110 269
-107 283
-94 279
-122 275
112 270
-88 289
133 269
146 264
-133 272
139 259
-113 268
25 308
144 261
128 268
-147 258
12 297
6 297
-72 287
104 271
-127 271
-105 277
-105 271
30
32 300
66 289
-83 281
-98 275
-25 290
133 267
35 293
0 291
19 296
90 293
69 293
-50 302
71 304
81 288
-15 294
14 293
-4 291
112 276
-90 293
103 279
10 307
-119 266
47 300
-20 291
47 297
47 301
-110 274
19 306
85 285
-32 300
30
138 254
-57 295
150 254
-140 261
86 289
-123 271
-69 291
-64 285
113 284
129 264
68 292
128 274
57 290
115 275
-2 305
6 304
112 274
87 286
-119 282
-72 303
-102 295
134 271
-4 295
-144 265
-127 277
15 296
63 300
49 300
34 299
42 300
30
144 272
16 292
-114 282
-104 292
85 287
47 302
-89 284
-140 268
26 306
-110 280
55 301
88 291
-47 302
68 285
139 275
64 300
-81 295
133 282
76 286
128 261
-114 281
34 294
-83 275
115 278
-5 292
-13 295
-113 278
-19 295
-41 303
143 261
30
-54 298
-6 296
-145 254
-1 298
-25 307
-146 256
74 279
-35 293
27 304
-88 290
-53 300
-85 290
17 294
130 271
-143 259
99 271
-54 295
54 285
-102 284
-47 298
-37 294
64 295
24 298
-33 304
-75 289
-53 298
98 287
-28 302
-124 276
94 281
