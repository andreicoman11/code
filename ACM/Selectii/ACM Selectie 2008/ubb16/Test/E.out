37371.55
41765.00
41469.40
34184.32
40470.76
45768.95
45301.00
43101.24
40512.82
40672.28
41617.26
32330.99
36883.97
43366.25
45463.76
39072.01
32794.84
43278.50
39394.99
40522.83
43308.64
45135.15
39557.39
39872.70
42988.57
43075.14
31752.50
43278.50
41762.04
38361.86
